dali/test/dali_test_single_op.h

```diff
@@ -330,10 +332,10 @@ class DALISingleOpTest : public DALITest {
       auto name = param.m_Name;
       switch (param.type) {
```

Comment:
Why didn't you use DALIDatatype here?

Reply:
Good question! I did not think about that. DONE.